### DISCUSSION TITLE
Laravel hasin

### DISCUSSION THREAD

**Discussion by biiiiiigmonster** [HassanZahirnia/laravel-package-ocean#106] [Package Suggestions]
[Package link](https://github.com/biiiiiigmonster/hasin)

Improve `has` or `whereHas` performance in Laravel Model.

**Comment by HassanZahirnia** [comment]
Thanks for the suggestion 😊 I've added the package!

**Reply by biiiiiigmonster** [reply]
Much appreciated!
